libs/MobileSync/src/com/salesforce/androidsdk/mobilesync/target/RefreshSyncDownTarget.java

```diff
@@ -57,49 +57,33 @@ public class RefreshSyncDownTarget extends SyncDownTarget {
     public static final String FIELDLIST = "fieldlist";
     public static final String SOBJECT_TYPE = "sobjectType";
     public static final String SOUP_NAME = "soupName";
-    public static final String COUNT_IDS_PER_SOQL = "coundIdsPerSoql";
+    public static final String COUNT_IDS_PER_SOQL = "countIdsPerSoql";
```

Comment:
Fixing this typo means that existing refresh sync will run with the default count ids per soql after upgrade. Shouldn't be a big deal since few are using that sync down target in the wild and even fewer if any have overridden the count ids per soql.